selection_sort/__init__.py

```diff
@@ -0,0 +1,20 @@
+def find_smallest(arr):
+    smallest_index = 0
+    smallest = arr[smallest_index]
```

Comment:
You can omit `smallest` and use only `smallest_index`

Reply:
`smallest` was necessary for comparison with value of `arr[i]`